packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -48,17 +48,15 @@ export function createAuthMiddleware({
       if (!access) {
```

Comment:
Do we still want to keep the  `dev-access-token` bypass (that's just above on L32?)

Reply:
Should we keep `dev-access-token` in general @mkurapov @wilsonianb ?

Reply:
We should remove it since I don't think it works: we would end up throwing in `httpsigMiddleware`

https://github.com/interledger/rafiki/blob/4e7cf597ac3de8f492025473a5839ca79bcf1f88/packages/backend/src/open_payments/auth/middleware.ts#L79-L88

Reply:
Yeah, let's make a separate issue for that @mkurapov

Reply:
https://github.com/interledger/rafiki/issues/906